src/providers/hive/auth.js

```diff
@@ -125,13 +126,18 @@ export const login = async (username, password) => {
 };
 
 export const loginWithSC2 = async (code) => {
-  const scTokens = await getSCAccessToken(code);
-  await hsApi.setAccessToken(get(scTokens, 'access_token', ''));
-  const scAccount = await hsApi.me();
-  const account = await getUser(scAccount.account.name);
-  let avatar = '';
 
-  return new Promise(async (resolve, reject) => {
+  try {
+    const scTokens = await getSCAccessToken(code);
+    hsApi.setAccessToken(get(scTokens, 'access_token', ''));
+    const scAccount = await hsApi.me();
+    
+    //NOTE: even though sAccount.account has account data but there are certain properties missing from hsApi variant, for instance account.username
```

Comment:
@noumantahir `account.username` is same as `account.name`

Reply:
essentially username is not always confirm to be present, that is one loophole I detected when I was able to reproduce the missing local issue. name property however is coming from hive itself, so it;s always certain to be there.